components/footer.js

```diff
@@ -5,10 +5,11 @@ import * as colors from '../styles/colors'
 
 export default () => {
   return (
-    <glamorous.Footer position="fixed" bottom={0} right={0} left={0}>
+    <glamorous.Footer bottom={0} right={0} left={0}>
```

Comment:
Maybe let's refactor this to match the same factory syntax in the `docs.js` file?